### PR TITLE
Fixed rapidnode to accept activationkey

### LINKED ISSUE #16: Accept activation key as input parameter
Right now rapidnode accepts content view as input.  It makes more sense for rapidnode to accept an activation key.  Change the input parameter from content view to activation key.

### PR DESCRIPTION
**Pull request by sthirugn**
This closes #16

### CODE REVIEW

**Comment by Ichimonji10** [comment]
ACK

**Comment by sthirugn** [comment]
@cswiii I looked at the user guide, we do call out registering through env/cv.  I am going to put that option back again.  I will add both content view and activation key option and the user can choose between the two

**Comment by cswiii** [comment]
Might as well remove the sample repo file from the product too, since we're removing the code used to upload it. Otherwise, ACK

**Comment by sthirugn** [comment]
@cswiii comments updated

**Comment by cswiii** [comment]
ACK

File: README.md [hunk 1]
```diff
@@ -26,7 +27,11 @@ A Satellite 6 system must be available, and the following should be true:
 3. A content view exists and has been published. It should provide the
    repositories from the previous step.
 4. An activation key exists. It should provide the content view from the
-   previous step. (FIXME: necessary?)
+   previous step.
+Note: Please choose between entering a content view or an activation key.  If
```
Comment (Ichimonji10) [inline README.md]: Add a blank line between lines 30 and 31.

File: katello_rapidnode.py [hunk 2]
```diff
@@ -9,24 +9,21 @@
...
 import paramiko
 
-REPO_FILE = 'myrepofile.repo'
 CONFIG = ConfigParser()
 CONFIG.read('katello_rapidnode.ini')
 PARENT = CONFIG['servers']['parent']
 
 
 def main():
+    "Main function to setup capsule(s)"
```
Comment (Ichimonji10) [inline katello_rapidnode.py]: Use triple quotes when creating a docstring.

    """Main function to set up capsule(s)."""

Also, there's no great need to write "Main function", as this is by definition a function called "main." It's just as informative to write something like this:

    """Set up one or more capsules."""

File: katello_rapidnode.py [hunk 3]
```diff
@@ -138,16 +135,25 @@ def child_register(parent, child):
...
         .format(parent)
     )
-    commands.append(
-        'subscription-manager register --username admin --auto-attach --force '
-        '--password {0} --org {1} --environment {2} '
-        .format(
-            CONFIG['credentials']['adminpassword'],
-            CONFIG['mainprefs']['orgname'],
-            CONFIG['mainprefs']['contentview'],
+    if CONFIG.get('mainprefs', 'contentview', fallback=None) is not None:
```
Comment (Ichimonji10) [inline katello_rapidnode.py]: Could this be written more simply as `if 'contentview' in CONFIG['mainprefs']:`?

File: katello_rapidnode.py [hunk 4]
```diff
@@ -138,16 +135,25 @@ def child_register(parent, child):
...
+        commands.append(
+            'subscription-manager register --username admin --auto-attach '
+            '--force --password {0} --org {1} --environment {2} '
+            .format(
+                CONFIG['credentials']['adminpassword'],
+                CONFIG['mainprefs']['orgname'],
+                CONFIG['mainprefs']['contentview'],
+            )
+        )
+    if CONFIG.get('mainprefs', 'activationkey', fallback=None) is not None:
```
Comment (Ichimonji10) [inline katello_rapidnode.py]: Could this be written more simply as `if 'activationkey' in CONFIG['mainprefs']:`?

**Comment by Ichimonji10** [comment]
ACK pending comments

File: katello_rapidnode.py (hunk 3 above)
Comment (sthirugn) [inline katello_rapidnode.py]: I prefer the existing approach which is more readable.  Also in future I may plan to extend the use of `get` by providing fallback values
Ref: https://docs.python.org/2/library/configparser.html#ConfigParser.ConfigParser.get

File: katello_rapidnode.py (hunk 4 above)
Comment (sthirugn) [inline katello_rapidnode.py]: same as above

File: katello_rapidnode.py (hunk 2 above)
Comment (sthirugn) [inline katello_rapidnode.py]: Yeah :) my intention was to say "to set up capsules". I will fix this

File: README.md (hunk 1 above)
Comment (sthirugn) [inline README.md]: sure

**Comment by sthirugn** [comment]
@Ichimonji10 Thank you. I addressed the doc comments.

**Pull request merged**